marge/job.py

```diff
@@ -63,7 +63,7 @@ def execute(self):
             log.exception('Unexpected Git error')
             merge_request.comment('Something seems broken on my local git repo; check my logs!')
             raise
-        except Exception:
+        except Exception as _ex:
```

Comment:
@aschmolk Why this change?

Reply:
Just for debugging, it's convenient to have the exception around.